system/settings/rbp2.xml

```diff
@@ -32,6 +32,11 @@
           <default>30</default> <!-- AE_QUALITY_MID -->
         </setting>
       </group>
+      <group id="1">
```

Comment:
I don't believe you need this change in rbp2.xml as rbp.xml will always be read first. rbp2.xml is just supplemental 

https://github.com/xbmc/xbmc/blob/master/xbmc/settings/Settings.cpp#L550-L553

Reply:
That's a very good observation :) I'll change it right away. Thanks.